src/main/osd/osd_elements.c

```diff
@@ -599,7 +603,54 @@ static void osdElementArtificialHorizon(osdElementParms_t *element)
         }
     }
 
-    element->drawElement = false;  // element already drawn
+   float earthUpinBodyFrame[3] = {-rMat[2][0], -rMat[2][1], -rMat[2][2]}; //transforum the up vector to the body frame
+   int thetaB; // pitch from body frame to up vector
+   int psiBSign; // psi from body frame to up vector
+   char arrowDir; // direction to draw arrow
+   float arrowDirFloat;
+   if(attitude.values.pitch>0.0){ //nose down
```

Comment:
Testing sign of `z` of up vector  may be cleaner.
Also, isn't it possible to select displayed vector (up/down vector) and unify other calculations?